framework/src/components/storage/entities/transaction.js

```diff
@@ -121,6 +121,81 @@ const BaseEntity = require('./base_entity');
 /**
  * Transaction Filters
  * @typedef {Object} filters.Transaction
+ * @property {string} [id]
+ * @property {string} [id_eql]
+ * @property {string} [id_ne]
+ * @property {Array.<string>} [id_in]
+ * @property {string} [id_like]
+ * @property {string} [blockId]
+ * @property {string} [blockId_eql]
+ * @property {string} [blockId_ne]
+ * @property {Array.<string>} [blockId_in]
+ * @property {string} [blockId_like]
+ * @property {string} [blockHeight]
```

Comment:
shouldn't this be integer? If so maybe you need to match all the types of the filters?

Reply:
@pablitovicente  The field is defined as a string with filter type `number`.
```
this.addField('blockHeight', 'string', {
	filter: filterTypes.NUMBER,
	fieldName: 'height',
});
```

Reply:
Hmm it's confusing what the `string` parameter means and what it affects for a field. It would be nice to document it whenever we have bandwidth.